### DISCUSSION TITLE
Clarity around `lr-` prefixed config files?

### DISCUSSION THREAD

**Discussion by vwb** [GoogleChrome/lighthouse#11986] [Q&A] [answered]
Hello!

I'm using lighthouse programmatically to generate reports on a running schedule, and digging into proper configuration. 

When looking through the lighthouse config files, I noticed that there are desktop and mobile configurations prefixed with `lr-`. Looking at the values of each, they are close to the default mobile and desktop. However, I am unable to determine what the significance of the `lr-` is, if there is any at all. 

(Ideal for programmatic running? What is used in pagespeed insights? etc)

Thanks!

**Comment by connorjclark** [comment, marked as answer]
They are use for Lightrider, which powers PSI.

**Reply by vwb** [reply]
Gotcha! Thank you for clearing that up, appreciate it. 

So if I were trying to configure programmatic runs to be as close as possible to the scores returned from PSI, would leveraging those configs would be the way to go?
